src/foam/dao/EasyDAO.js

```diff
@@ -189,6 +222,11 @@ return delegate;
       type: 'foam.dao.DAO',
       name: 'innerDAO'
     },
+    {
+      class: 'Object',
+      type: 'foam.dao.DAO',
+      name: 'decorator'
```

Comment:
This would be more useful if there were more than one.

Reply:
@kgrgreer It handles a ProxyDAO - walks the chain and sets the delegate on the last.